docs/guide/database-queries.md

```diff
@@ -128,12 +143,13 @@ One very useful query builder is `.project()`. We often know what specific data
 query
   .project({
     title: 1,
+    _url: 1,
     _author: 1,
     publishedAt: 1
   })
 ```
 
-The `project` builder is one that uses a MongoDB syntax. The object passed as an argument should include schema field names for the document type we're querying with each set to `1`. This tells the database that we only want these fields in each document we get back.
+The `project` builder is one that uses a MongoDB syntax. The object passed as an argument should include schema field names for the document type we're querying with each set to `1`. This tells the database that we only want these fields in each document we get back. In addition to simple fields, Apostrophe enhances `project` to handle relationship fields and the special property `_url`.
```

Comment:
Any truthy value will do but `1` is the convention, yeah.

Reply:
I'll say `1` or `true` since that's what MongoDB docs list. Even if other truthy values will work I don't think it helps devs to carry any more than this in their head. https://docs.mongodb.com/manual/reference/method/db.collection.find/#projection